modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -25,34 +25,60 @@ static IP address (`ip=<host_ip>`). Then identify the DNS server IP address (`na
 * The hostname to `core0.example.com` +
 * The DNS server address to `4.4.4.41`
 
-|`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none nameserver=4.4.4.41` +
+a|
+----
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
+nameserver=4.4.4.41
+----
 
 |Specify multiple network interfaces by specifying multiple `ip=` entries.
-|`ip=192.168.122.25 ip=192.168.122.26`
+a|
+----
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none
```

Comment:
I would use a different IP address here on the second interface to demonstrate the use of multiple IPs

Reply:
Of course. Darn. Dangers of copy and paste. It's fixed now. I'm going to squash and merge. So, unless I hear otherwise, I expect it is ready to publish.